test/expected/query-15.out

```diff
@@ -334,33 +360,45 @@ BEGIN;
 
   :PREFIX SELECT time_bucket(10, time) t, avg(series_0), min(series_1), avg(series_2)
   FROM hyper_1_int GROUP BY t ORDER BY t DESC limit 2;
-                                          QUERY PLAN                                          
-----------------------------------------------------------------------------------------------
+                                  QUERY PLAN                                   
```

Comment:
This looks like another regression, dont think Sort->HashAgg->SeqScan is better then OrderedAppend->IndexScan

Reply:
Depends on how much the cardinality is reduced after grouping. I think I'm going to target this in this PR https://github.com/timescale/timescaledb/pull/7025
Basically it's going to take the cheapest plan as a basis for chunk-wise aggregation, so in this case it will be OrderedAppend over Partial GroupAggregates over IndexScans.